### ISSUE TITLE
Feature Introduction Page

### ISSUE THREAD

**Issue by EMLVIRUS** [bwinton/VerticalTabs#283]
Add a built-in or online page (wiki page on this git page is also fine) for tab center.

**Comment by terrycloth** [comment]
I agree this would be helpful. For example: I've been using Tab Center for a while. I didn't realize that the pinned-open side bar could be resized until I came here to make a feature request, and then discovered Issues #361 and #23. Pinning and/or resizing the side bar ought to be part of the Tab Center tutorial, and probably ought to display on first installation, the way Firefox itself walks the user through a few basic features on first run.
